### ISSUE TITLE
figure out if/how we can remove test releases from the CHANGELOG.md

### ISSUE THREAD

**Issue by surchs** [neurobagel/planning#103]

**Comment by surchs** [comment]
@alyssadai haven't you already completed this issue?

**Comment by alyssadai** [comment]
@surchs I don't think so - I think by default duplicate releases are not removed/overwritten in the changelog. e.g., see https://github.com/neurobagel/bagel-cli/blob/main/CHANGELOG.md which has multiple of the same version from when I was testing out releasing using the Neurobagel Bot PAT.

I'm not sure there's an easy or auto way to do this, so we may just want to delete them manually to clean up the CHANGELOGs. Now that we have a pretty good release wf, hopefully this won't happen again in the future

**Comment by surchs** [comment]
I see, thanks for clarifying!
